src/main/java/frc/robot/Robot.java

```diff
@@ -266,12 +266,26 @@ private boolean isAllianceRed() {
         return allianceOptional.isPresent() && allianceOptional.get() == DriverStation.Alliance.Red;
     }
 
-    private boolean isCommandIsValid(double length, double angle, double distance) {
+    private double getXDistance(double targetAngle, double armTargetLength, double clawTargetAngle) {
```

Comment:
what does `distance` have to do with this? What the hell is the calculation here?